docker/cygnus-ngsi/Dockerfile

```diff
@@ -16,7 +16,7 @@
 # For those usages not covered by the GNU Affero General Public License please contact with iot_support at tid dot es
 #
 
-FROM centos:centos7.6.1810
+FROM centos:centos7.9.2009
```

Comment:
I think we have several Dockerfiles in this repo (four?). I'd suggest to do the same fix in all them.